site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,409 @@
+
+For a long time, if you wanted to add cheats in-game, you needed to do this manually via editing a text file containing the cheat code(s) in the [Citra User Directory](https://citra-emu.org/wiki/user-directory/). This proved to be a complicated way to do things, especially when taking Android’s file management systems into consideration.
+
+Fans of Citra had been begging for an intuitive way to add cheats to their favorite 3DS games. [JosJuice](https://github.com/JosJuice), a developer for the GameCube and Wii emulator [Dolphin](https://dolphin-emu.org/), stepped up to the task and added a cheat GUI based off of Dolphin’s own implementation! 
+
+A hurdle in development was found instantly, though. If you slid a panel away whilst focused on a text box, such as the cheats text box, the keyboard would stay open without an option to close it! This would be frustrating, and require a restart of Citra to get rid of the keyboard.
+With a fix ported over from Dolphin, originally written by [t895](https://github.com/t895), the keyboard now closes automatically, as intended!
+
+The new cheat menu allows you to input the cheat name, the cheat itself, and any notes about the cheat you would like to add! The cheat menu can be accessed from the in-emulation menu to add, modify, and delete your cheats. However, cheats cannot be disabled whilst in-game. You must reboot your game for any changes to the cheat code to take effect.
+
+{{< figure src="cheats.png"
+    title="Getting more money could not be any easier!" >}}
+
+## Add support for custom textures and texture dumping ([#6144](https://github.com/citra-emu/citra/pull/6144)) by [LeviathaninWaves](https://github.com/LeviathaninWaves)
+
+Custom Textures are a way for modders to revamp the textures of a game to make them even more amazing than ever before! Due to the low resolution of the 3DS, games often used low-quality, compressed images for the models, so when Citra renders at upscaled resolutions, it’s still just not quite perfect.
+Texture Dumping is a way to gather all textures from a game whilst playing through it. No need to do anything fancy, all you need to do is play! These can be used for archival purposes or to create your own Custom Texture Packs to add right back into Citra.
+
+First [added to Citra Desktop](https://github.com/citra-emu/citra/pull/4868) all the way back in November 2019, first time developer [LevithaninWaves](https://github.com/LeviathaninWaves) tasked themselves with implementing this feature over on our Android build.
+
+To add Custom Textures to Citra, navigate to `Internal Storage -> citra-emu`, create a folder called `load`, another folder inside that called `textures` then a folder named with the TitleID of your game. This can be found on the [3DS releases database](http://3dsdb.com/) if you do not know it. 
```

Comment:
```suggestion
To add Custom Textures to Citra, navigate to `Internal Storage -> citra-emu`, create a folder called `load`, another folder inside that called `textures` then a folder named with the TitleID of your game. It can be found on the [3DS releases database](http://3dsdb.com/) if you do not know it. 
```

Reply:
"This TitleID" would maybe be better for clarity. Since the previous sentence contains a lot of sub-clauses.